documentation/tut/toy-example-from-scratch.rst

```diff
@@ -27,19 +29,65 @@ Below are the ``flexmeasures`` CLI commands we'll run, and which we'll explain s
 Okay, let's get started!
 
 
+.. note:: You can copy the commands by hovering on the top right corner of code examples. You'll copy only the commands, not the output!
+
 Install Flexmeasures and the database
 ---------------------------------------
 
-This example is from scratch, so we'll assume you have nothing prepared but a (Unix) computer with Python and two well-known developer tools, `pip <https://pip.pypa.io>`_ and `docker <https://www.docker.com/>`_
+.. tabs::
 
-We install the FlexMeasures platform, use Docker to run a postgres database and tell FlexMeasures to create all tables.
+  .. tab:: Docker
 
-.. code-block:: console
+        If `docker <https://www.docker.com/>`_ is running on your system, you're good to go. Otherwise, see `here <https://docs.docker.com/get-docker/>`_.
+
+        We start by installing the FlexMeasures platform, and then use Docker to run a postgres database and tell FlexMeasures to create all tables.
+
+        .. code-block:: console
+
+            $ docker pull lfenergy/flexmeasures:latest
+            $ docker pull postgres
+            $ docker run --rm --name flexmeasures-tutorial-db -e POSTGRES_PASSWORD=docker -e POSTGRES_DB=flexmeasures-db -d -p 5433:5432 postgres:latest 
```

Comment:
In the docker tab, we're naming the database `flexmeasures-db`, while in the PC tab, we're naming it `flexmeasures`. Let's sync that, unless you have a reason not to. Same goes for the password, the port and `localhost` vs `127.0.0.1`.

Reply:
Mostly agree, but in the docker tab, the port is changed to 5433, so as not to collide with already running Postgres instances on the host PC.